suite-native/app/app.config.ts

```diff
@@ -145,8 +145,10 @@ export default ({ config }: ConfigContext): ExpoConfig => {
             backgroundColor: '#25292E',
             resizeMode: 'contain',
         },
+        userInterfaceStyle: 'automatic',
         android: {
             package: bundleIdentifier,
+            userInterfaceStyle: 'automatic',
```

Comment:
I don't think it's necessary to duplicate this property for each platform if you already set it in general for both...